features/step_definitions/step_definitions.rb

```diff
@@ -0,0 +1,89 @@
+Given(/^I am on the homepage$/) do
+  visit('/')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.